phpBB/includes/compatibility_globals.php

```diff
@@ -44,7 +44,6 @@
 // Grab global variables, re-cache if necessary
 /* @var $config phpbb\config\db */
 $config = $phpbb_container->get('config');
-set_config(null, null, null, $config);
```

Comment:
The `$config` injection was used in three main places:
* The board itself
* The install
* In tests

Now it is no longer used at all. So maybe it would be easier to use `global $config;` and use the global config object? I don't know if some developers used the config injection.

For `request_var()` is still used in tests, but that's it. So I think it should use the same `global $request;`, no?